DOC-11021  starter kits (#4056)
couchbase/docs-server: modules/getting-started/pages/starter-kits.adoc

++++
endif::[]

= Starter Kits

> Couchbase's https://www.couchbase.com/developers/[Developer Site] contains several useful https://developer.couchbase.com/tutorials/[Quickstart Guides].

Reviewer: Just wondering why the link text says "Quickstart Guides" but the link is for tutorials?

According to the drop-down filters on that page, the quickstart guides are a sub-type of tutorials. There are 30 quickstart guides out of a total of 159 of all types.

```suggestion
Couchbase's https://www.couchbase.com/developers/[Developer Site] contains several useful https://developer.couchbase.com/tutorials/[Tutorials].
```

These tutorials cover developing for Couchbase, using Couchbase SDKs, within the context of a wider software ecosystem -- such as developing with the Java SDK, using Spring Data.

> Go to the https://developer.couchbase.com/tutorials/[Developer Quickstart Guides] or the 

Reviewer: Again, why Quick Start Guides rather than tutorials?

Also, rather than just repeating the link, maybe deep-link at this point to the list of tutorials within the page

```suggestion
Go to the https://developer.couchbase.com/tutorials/#cb-tutorials-container-header[Developer Tutorials] or the 
```

Author: Yeah, we originally had this page just for Quickstart guides - and my wording reflects that, but it no longer seems so correct.
Let's go with tutorials :+1:

The seatback web app accesses Edge Server via a RESTful interface.
When there is Internet connectivity, the Edge Server syncs data with remote Capella App Services.

https://github.com/couchbase-examples/edge-server-meal-order-sample-app[Edge Server & TypeScript]

[.column]
====== {empty}
.iOS Swift P2P Quickstart

[.content]
This quick-start app demonstrates how to use Couchbase Lite Swift to load a prebuilt database, create and update documents, and query documents.
It also shows how to set up peer-to-peer replication using the Multipeer Replicator, which leverages DNS-SD (Bonjour) for automatic peer discovery over Wi-Fi, enabling synchronization between nearby devices without requiring a centralized server.
[]
https://github.com/couchbase-examples/ios-swift-quickstart-p2p[iOS Swift P2P Quickstart]

[.column]
====== {empty}
.React Native

[.content]
Expo-based React Native app that uses `cbl-reactnative` and the Travel Sample bucket dataset with the hotels and landmark collections.

https://github.com/couchbase-examples/expo-cbl-travel[React Native Expo Travel Sample App]

+++
</div>
+++

[.column]
====== {empty}

> == Agentic Apps, LLM, & RAG

Reviewer: Should these be first on the page? (New hotness)

Author: Good idea!

++++
<div class="card-row two-column-row">
++++

[.column]
====== {empty}
.Agent Catalog Quickstart

[.content]
This repository provides a quickstart guide for using the Agent Catalog with Capella Model Services and Couchbase.
[]
https://github.com/couchbase-examples/agent-catalog-quickstart[Agent Catalog Quickstart]

[.column]
====== {empty}
.Chatbot Demo

[.content]
Q&A Chatbot Demo using Couchbase, `LangChain`, `OpenAI`, and `Streamlit`.
[]
https://github.com/couchbase-examples/qa-bot-demo[Chatbot Demo]

+++
</div>
+++

== {empty}

++++
<div class="card-row two-column-row">
++++

[.column]
====== {empty}

> .Llamaindex RAG Demo

Reviewer: Capitalization?

```suggestion
.LlamaIndex RAG Demo
```
